yorkie/src/main/kotlin/dev/yorkie/document/Document.kt

```diff
@@ -451,6 +451,10 @@ public class Document(
         onlineClients.value -= actorID
     }
 
+    internal fun setStatus(newStatus: DocumentStatus) {
```

Comment:
do we need additional setter method for this?
Kotlin's properties are different from Java's fields, and it is not wrong to use properties for simple getter/setter's.
(https://blog.kotlin-academy.com/kotlin-should-i-define-function-or-property-6786951da909)

Reply:
I thought setting the `Document.status` from `Client` directly can violate the encapsulation. I reverted it based on your suggestion: [8febcf6](https://github.com/yorkie-team/yorkie-android-sdk/pull/186/commits/8febcf6c7fdf75aa47a009bf59db7f22a756bc62)